lib/pages/home.dart

```diff
@@ -1,170 +1,170 @@
+            title: new Text('GitHub Grass'),
+            actions: <Widget>[
+              new IconButton(
+                icon: new Icon(Icons.refresh),
+                onPressed: model.update,
+              )
+            ],
+          ),
+          body: new ListView.builder(
+            itemCount: monthlyContributions.length + 1,
+            itemBuilder: (BuildContext context, int index) {
+              if (index == 0) {
+                return _createIdentityWidget();
+              } else {
+                _ContributionTile monthlyContrib =
+                    monthlyContributions[index - 1];
+                return _createMonthlyContributionMap(monthlyContrib);
+              }
+            },
+          ));
+    }
+
+    Widget _createIdentityWidget() {
+      DateTime today = new DateTime.now();
+
+      Contribution c = model.contributions.firstWhere((c) =>
+          c.year == today.year && c.month == today.month && c.day == today.day);
+      int value = c.value ?? 0;
+
+      String todayLabel =
+          '${today.year.toString()}-${today.month.toString().padLeft(2,'0')}-${today.day.toString().padLeft(2,'0')}';
+
+      return new Padding(
+        padding: const EdgeInsets.symmetric(vertical: 10.0),
+        child: new Center(
+          child: new Column(
+            children: <Widget>[
+              new CircleAvatar(
+                backgroundImage: new NetworkImage(
+                    'https://avatars.githubusercontent.com/${model.username}'),
+              ),
+              new Text(
+                model.username,
                 style: new TextStyle(
-                  fontSize: 10.0,
+                  fontSize: 20.0,
                 ),
-              );
-              valueText = _createValueBox(c);
-              return new Flexible(
-                flex: 1,
-                child: monthlyContrib.isPresentMonth()
-                    ? valueText
-                    : new Center(
-                        child: valueText,
+              ),
+              value > 0
+                  ? new Text(
+                      'Today\'s contribution is ${value}! 👍',
+                      style: new TextStyle(
+                        color: Colors.green,
+                        fontWeight: FontWeight.bold,
                       ),
-              );
-            },
-          ).toList(),
-        )
-      ],
-    );
-  }
-
-  Widget _createValueBox(int value) => new Container(
-        height: 20.0,
-        color: (value == 0)
-            ? Colors.grey[200]
-            : Colors.green[(value > 8 ? 9 : value) * 100],
+                    )
+                  : new Text(
+                      '${todayLabel} is not contributed yet.❌',
+                      style: new TextStyle(
+                        color: Colors.red,
+                      ),
+                    ),
+            ],
+          ),
+        ),
       );
-}
+    }
+
+    Widget _createMonthlyContributionMap(_ContributionTile monthlyContrib) {
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Use => for short members whose body is a single return statement.